docs/contributors.yaml

```diff
@@ -777,3 +777,9 @@
     - xingzhou@cn.ibm.com
   jira_user: dongdong
   reviewboard_user: dongdong
+  
+- name: Antonis Danezis
```

Comment:
Can you sort the entry based on first name? Thanks!

Reply:
Ah my bad, didn't realize they were sorted. Hope it's fixed now